wls-gui-wahllokalsystem/tests/components/common/cards/BaseInputFeedbackCard.spec.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  COMPONENT_RENDER_TESTS,
+  getSnapshotFilename,
+} from "@tests/utils/testutils.ts";
+import { mount, VueWrapper } from "@vue/test-utils";
+import { afterEach, describe, expect, it, vi } from "vitest";
+
+import BaseInputFeedbackCard from "@/components/common/cards/BaseInputFeedbackCard.vue";
+import vuetify from "@/plugins/vuetify.ts";
+import { InputFeedbackTypeEnum } from "@/types/common/InputFeedbackTypeEnum.ts";
+
+const mockDefinitions = vi.hoisted(() => ({
+  getBorderColorForInputFeedbackType: vi.fn(),
+  getIconColorForInputFeedbackType: vi.fn(),
+  getIconForInputFeedbackType: vi.fn(),
+  getTextColorForInputFeedbackType: vi.fn(),
+}));
+
+vi.mock("@/composables/common/inputFeedbackUtils.ts", () => ({
+  useInputFeedbackUtils: () => ({
+    getBorderColorForInputFeedbackType:
+      mockDefinitions.getBorderColorForInputFeedbackType,
+    getIconColorForInputFeedbackType:
+      mockDefinitions.getIconColorForInputFeedbackType,
+    getIconForInputFeedbackType: mockDefinitions.getIconForInputFeedbackType,
+    getTextColorForInputFeedbackType:
+      mockDefinitions.getTextColorForInputFeedbackType,
+  }),
+}));
+
+describe("BaseInputFeedbackCard.vue", () => {
+  let wrapper: VueWrapper;
+
+  afterEach(() => {
+    vi.clearAllMocks();
+  });
+
+  describe(COMPONENT_RENDER_TESTS, () => {
+    it("should_renderWithoutAdditionalFeedback_when_additionalFeedbackSlotIsNotUsed", async (context) => {
+      const feedbackType = InputFeedbackTypeEnum.error;
+
+      mockDefinitions.getBorderColorForInputFeedbackType.mockReturnValue(
+        "mockedBorderColor"
+      );
+      mockDefinitions.getIconColorForInputFeedbackType.mockReturnValue(
+        "mockedIconColor"
+      );
+      mockDefinitions.getIconForInputFeedbackType.mockReturnValue(
+        "$information"
```

Comment:
```suggestion
        "$error"
```
ist zwar nur ein test aber hat mich verwirrt weil du oben feedbacktype error definiert hast dann wüde ich den rest auch passend dazu machen. bitte in beiden tests anpassen

Reply:
angepasst mit 7650d0f3b3482529bec8c3ca755513833aae2a63